Sources/Event Logging/TracksService.m

```diff
@@ -1,24 +1,32 @@
 #import "TracksService.h"
 #import "TracksDeviceInformation.h"
 #import "TracksLoggingPrivate.h"
-#import <Reachability/Reachability.h>
+#import <Network/Network.h>
+
+#if SWIFT_PACKAGE
+@import AutomatticTracksModel;
+@import AutomatticExperiments;
+@import CocoaLumberjack;
+#else
 #import <AutomatticTracks/AutomatticTracks-Swift.h>
+#endif
+
 
 #if TARGET_OS_IPHONE
 #import <UIKit/UIKit.h>
 #else
 #import <AppKit/AppKit.h>
 #endif
 
-
 @interface TracksService ()
 
 @property (nonatomic, strong) NSTimer *timer;
 @property (nonatomic, assign) BOOL timerEnabled;
 @property (nonatomic, assign) BOOL isHostReachable;
 @property (nonatomic, strong) TracksContextManager *contextManager;
 @property (nonatomic, strong) TracksDeviceInformation *deviceInformation;
-@property (nonatomic, strong) Reachability *reachability;
+@property (nonatomic, strong) nw_path_monitor_t networkMonitor;
+@property (nonatomic, strong) nw_path_t networkPath;
```

Comment:
As I mentioned above, Reachability is deprecated, and `nw_path_monitor` is the modern way to accomplish the same functionality.